modules/MultiDRM/MultiDRM.loader.js

```diff
@@ -140,7 +140,7 @@
 			var hlsSource = sources.filter( function ( source ) {
 				return ( source.mimeType === "application/vnd.apple.mpegurl" );
 			});
-			drmSources.push(hlsSource);
+			drmSources.concat(hlsSource);
 		}
```

Comment:
Thats exactly what I meant. Never mind - merging, we can change the name later if we want...